architectury-reflection-redirector/src/main/java/dev/architectury/reflectionredirector/ArchitecturyReflectionRedirectorPlugin.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * This file is licensed under the MIT License, part of architectury-loom-runtime.
+ * Copyright (c) 2020-2023 architectury
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package dev.architectury.reflectionredirector;
+
+import cpw.mods.modlauncher.serviceapi.ILaunchPluginService;
+import net.fabricmc.mappingio.MappingReader;
+import net.fabricmc.mappingio.tree.MappingTree;
+import net.fabricmc.mappingio.tree.MemoryMappingTree;
+import org.objectweb.asm.Type;
+import org.objectweb.asm.tree.ClassNode;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.UncheckedIOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.EnumSet;
+
+public class ArchitecturyReflectionRedirectorPlugin implements ILaunchPluginService {
+    public static ArchitecturyReflectionRedirectorPlugin instance;
+    public static final EnumSet<Phase> AFTER_ONLY = EnumSet.of(Phase.AFTER);
+    // Namespaces in mapping file
+    private static final String TARGET_NAMESPACE = "named";
+
+    private static final String MAPPINGS_PATH_PROPERTY = "architectury.reflectionredirector.mappingsPath";
+    private static final String SOURCE_NAMESPACE_PROPERTY = "architectury.reflectionredirector.sourceNamespace";
```

Comment:
```suggestion
    private static final String MAPPINGS_PATH_PROPERTY = "architectury.reflectionRedirector.mappingsPath";
    private static final String SOURCE_NAMESPACE_PROPERTY = "architectury.reflectionRedirector.sourceNamespace";
```
Nit: should be camelcase